### ISSUE TITLE
rgamma function in simmer: increasing scale parameter

### ISSUE THREAD

**Issue by IoannaChr** [r-simmer/simmer#236]
I have the following code where I am calculating the waiting time of ships. I try to increase the scale of the barge trajectory (in rgamma) in order to decrease the waiting times while I decrease the arrivals of barges ( first input in rgamma in add_generator of barges). However, when I increase the scale while decreasing the arrivals then the waiting times are increasing instead of decreasing. They are only decreasing if I decrease the scale but it should be opposite. Am I entering something in the wrong way? In the initial calculations I use a scale of 0.09 (in the barge trajectory) and 609 arrivals (in the add_generator) whereas in the next stage (code below) I use a scale of 0.11382 (barge trajectory) and 483 arrivals (in the add_generator). Besides that I keep everything else constant. 
Thank you in advance!

```r
library(simmer)

set.seed(1269)

terminal <- simmer("terminal")

deepsea <-
  trajectory("Deep sea path") %>%
  log_("Here I am") %>%
  set_attribute("start_time", function() {now(terminal)}) %>%
  seize("berth") %>%
  timeout(function() {rgamma(1,shape=2.53,scale=0.4)}) %>%
  #  log_(function() {paste("Waited: ", now(terminal) - get_attribute(terminal, "start_time"))}) %>%
  release("berth") %>%
  log_(function() {paste("Finished: ", now(terminal))})


barge <-
  trajectory("Barge path") %>%
  log_("Here I am") %>%
  set_attribute("start_time", function() {now(terminal)}) %>%
  seize("berth") %>%
  #  log_(function() {paste("Waited: ", now(terminal) - get_attribute(terminal, "start_time"))}) %>%
  timeout(function() {rgamma(1, shape=1.24,scale=0.11382)}) %>%
  release("berth") %>%
  log_(function() {paste("Finished: ", now(terminal))})

terminal <-
  simmer("terminal") %>%
  add_resource("berth",8,preemptive=TRUE) %>%
  add_generator("Deep sea vessel", deepsea, function() {c(0, rgamma(79, shape=1.47,scale=0.23))},priority=1,preemptible = 1, restart=TRUE) %>%
  add_generator("Barge", barge, function() {c(0, rgamma(483, shape=1.14,scale=0.04))},priority=0,preemptible=0, restart=TRUE) %>%
  run(until = 500)

scenario4_restart58=terminal %>%
  get_mon_arrivals() %>%
  transform(waiting_time = end_time - start_time - activity_time)%>%
  transform (start_service_time= start_time + waiting_time)

write.csv(scenario4_restart58,'scenario4_restart58.csv',row.names = F)
```

**Comment by Enchufa2** [comment]
You are thinking about the rate, but rate and scale are not the same. In fact, they have an inverse relationship. So, yes, when you increase the scale, the average value increases.

**Comment by IoannaChr** [comment]
I tried to increase the scale of barges (meaning that their service time will increase) while the amount of barges will be less. I didn't increase the rate instead because that would mean a decrease in scale meaning a decrease in service time which is not the case.  Or am I thinking wrong about the scale-rate? Scale is the theta (θ) of the gamma distribution so I changed it because I am changing the theta in my calculations.
